src/core/Akka/Actor/RepointableActorRef.cs

```diff
@@ -24,16 +24,16 @@ public class RepointableActorRef : ActorRefWithCell, IRepointableRef
         private readonly ActorSystemImpl _system;
         private readonly Props _props;
         private readonly MessageDispatcher _dispatcher;
-        private readonly Func<Mailbox> _createMailbox;
+        private readonly MailboxType _mailboxType;
```

Comment:
I need all that fields in RoutedActorRef, maybe we should make them as protected?